sql/core/pom.xml

```diff
@@ -92,6 +92,11 @@
       <version>${fasterxml.jackson.version}</version>
     </dependency>
     <dependency>
+      <groupId>com.ibm.icu</groupId>
```

Comment:
ICU is widely used, as shown in http://site.icu-project.org . A very useful package. Before, we used it for codepage conversion.